src/main/java/HttpServer/MockClient.java

```diff
@@ -0,0 +1,26 @@
+package HttpServer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class MockClient implements Writable {
```

Comment:
I very much like the `Writable` abstraction. It's very focused and isolates the writing of the line

Reply:
:heavy_check_mark: